content/financial-aid.html

```diff
@@ -3,6 +3,14 @@
 title: Financial Aid
 ---
 
+<h2>Apply Now</h2>
+
+<p>
+    To apply for financial aid, please fill out and submit <a "https://docs.google.com/forms/d/1JWdkK4pP-USm3duQ-mXdQ6rYEp05lrmqMQ0sSCA5JXw/viewform?embedded=true">this form</a>.
```

Comment:
`this form` doesn't appear as a hyperlink for me, you are missing the `href=` in the anchor tag. Also, you have `embedded=true` in the URL, but I'm not sure you are actually embedding the form?